numpy_cpp-custom-matrix/example.cpp

```diff
@@ -66,20 +66,18 @@ Matrix<double> Transpose(const Matrix<double> &a) {
   return ret;
 }
 
+double Get_exp(const double a) {
+  double ret = 1;
+  int terms = 1000;
+  for (int i = terms - 1; i > 0; --i )
+    ret = 1 + a * ret / i;
+  return ret;
+}
+
```

Comment:
You don't need to implement exp(). Please remove the function Get_exp() and use the C++ exp() function instead.